iac-common/src/test/java/org/sonar/iac/common/extension/IacSensorTest.java

```diff
@@ -71,6 +77,29 @@ void test_descriptor() {
     assertThat(sensorDescriptor.name()).isEqualTo("IaC Common Sensor");
   }
 
+
+  @Test
+  void test_descriptor_sonarqube_9_3() {
+    final boolean[] called = {false};
+    DefaultSensorDescriptor descriptor = new DefaultSensorDescriptor() {
+      public SensorDescriptor processesFilesIndependently() {
+        called[0] = true;
+        return this;
+      }
+    };
+    sensor(SonarRuntimeImpl.forSonarQube(Version.create(9, 3), SonarQubeSide.SCANNER, SonarEdition.DEVELOPER), null)
+      .describe(descriptor);
+    assertThat(called[0]).isTrue();
```

Comment:
This test method is weird to me. Could we use something like mockito to make its intent clearer?